pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "support-docs"
```

Comment:
I don't think it matters at all, but I would expect this to match the repository name.

Reply:
Fair, changed.